include/pybind11/chrono.h

```diff
@@ -161,9 +161,17 @@ template <typename Duration> class type_caster<std::chrono::time_point<std::chro
         // > If std::time_t has lower precision, it is implementation-defined whether the value is rounded or truncated.
         // (https://en.cppreference.com/w/cpp/chrono/system_clock/to_time_t)
         std::time_t tt = system_clock::to_time_t(time_point_cast<system_clock::duration>(src - us));
+
+        // std::localtime returns a pointer to a static internal std::tm object on success,
+        // or null pointer otherwise
+        // (https://en.cppreference.com/w/cpp/chrono/c/localtime)
```

Comment:
IMO, the link is not really needed, since we can find it back, when necessary. But it can stay if you prefer.

Reply:
I can remove it

Reply:
You don't per se have to; it's not wrong either! (I just thought cppreference.com is the go-to reference anyway :-) )